### PR TITLE
librbd: small fixes

### PR DESCRIPTION
**Pull request by xiexingguo**
Fix http://tracker.ceph.com/issues/14127
And improve some tip messages.

Signed-off-by: xie xingguo <xie.xingguo@zte.com.cn>

### CODE REVIEW

**Comment by xiexingguo** [comment]
@dillaman Some more small fixes about RBD module. Hope you wouldn't mind and take a look at.
Thanks a lot.

File: src/librbd/internal.cc [hunk 1]
```diff
@@ -256,6 +256,7 @@ int validate_pool(IoCtx &io_ctx, CephContext *cct) {
   if (r < 0) {
     lderr(cct) << "failed to release self-managed snapshot " << snap_id
                << ": " << cpp_strerror(r) << dendl;
+    return r;
```
Comment (dillaman) [inline src/librbd/internal.cc]: Not returning the error result was intentional.  There is no way to recover and delete the snapshot id ever again -- and the pool has already been changed to self-managed snapshot mode.  Therefore, no need to abort the create request.

**Comment by dachary** [comment]
@xiexingguo you can ignore the bot failure at http://jenkins.ceph.dachary.org/job/ceph/LABELS=centos-7&&x86_64/10346/, it is a known false negative ( see http://tracker.ceph.com/issues/14092 for details).

**Comment by xiexingguo** [comment]
OK, @dachary, thanks for your reminder. And you may need to check 10348 also.

**Comment by xiexingguo** [comment]
@dillaman Please take a look at this one too, which is fixed as just you suggested, thanks!

**Comment by xiexingguo** [comment]
@dillaman Is this one ready for merging? I feel liking asking as these are some quite simple cleanups.

**Pull request merged**